File: app/src/main/java/com/example/academygestormobile/Models/Recibo.kt
package com.example.academygestormobile.Models

import java.util.Date

data class Recibo(
    val id: Int,
    val matricula: Matricula,
    val detalle: String,
    val fecha: Date,
    val importe: Double,
    val descuento: Byte,
    val pagado: Byte
)

File: Apps/AcademyGestor/AcademyGestor/Models/Recibo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AcademyGestor.Modelos
{
    public class Recibo
    {
        public int id { get; set; }
        public Matricula matricula { get; set; }
        public string detalle { get; set; }
        public DateTime fecha { get; set; }
        public double importe { get; set; }
        public byte descuento { get; set; }
        public byte pagado { get; set; }
    }
}
